aws/resource_aws_ssm_association_test.go

```diff
@@ -958,6 +958,7 @@ DOC
 resource "aws_ssm_association" "test" {
   name                = aws_ssm_document.test.name
   schedule_expression = "cron(0 16 ? * TUE *)"
+  apply_only_at_cron_interval = true
```

Comment:
To ensure we are not introducing any regressions, going to split this new argument into its own testing on merge.